src/client/js/categoryData.js

```diff
@@ -0,0 +1,38 @@
+import { fetchCategory } from './api';
+
+export default class CategoryData {
+  #data = [];
+
+  constructor(){
+    this.#setState([
+      "JavaScript",
+      "NodeJs",
+      "NextJs",
+      "HTML5",
+      "CSS3",
+      "ReactJS",
+      "Redux",
+      "Redux-saga",
+			"VueJS",
+			"Vuex"
+    ]);
+
+    // TODO: 서버로 데이터 초기화
+    // this.#setState(this.#fetchData());
+  }
+
+  #setState(nextData) {
+    this.#data = nextData;
+  }
+
+  getData(){
+    return this.#data;
+  }
+
+  // async #fetchData(){
```

Comment:
위와 상동. TODO 처럼 필요한 부분을 제외한 불필요한 코드나 주석을 제거해주시고 올려주세요
```suggestion
  // async #fetchData(){
```